graylog2-web-interface/src/components/common/CustomMenuList.tsx

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
This component is so coupled to the `Select` component, let's create a `Select` directory and store this file and the `Select` component in it. The directory can contain an `index.ts` which exports the `Select` component. This way it will still be possible to import it from `component/common/Select`, while we still have a `Select.tsx` file.